policyengine_us/parameters/gov/states/va/tax/income/deductions/standard.yaml

```diff
@@ -8,10 +8,12 @@ metadata:
   reference:
   - title: Code of Virginia § 58.1-322.03.(1)(b)
     href: https://law.lis.virginia.gov/vacodefull/title58.1/chapter3/article2/
+  - title: Virginia 2023 Form 760 Resident Individual Income Tax Instructions
+    href: https://www.tax.virginia.gov/sites/default/files/vatax-pdf/2023-760-instructions.pdf#page=18
   - title: Virginia 2022 Form 760 Resident Individual Income Tax Booklet
     href: https://www.tax.virginia.gov/sites/default/files/vatax-pdf/2022-760-instructions.pdf#page=18
   - title: Virginia 2021 Form 760 Resident Individual Income Tax Booklet
-    href: https://www.tax.virginia.gov/sites/default/files/vatax-pdf/2021-760-instructions.pdf#page=19
+    href: https://www.tax.virginia.gov/sites/default/files/vatax-pdf/2021-760-instructions.pdf#page=18
```

Comment:
could you add the 2024 - 2026 standard deduction changes while in here?
[Link](https://www.tax.virginia.gov/sites/default/files/inline-files/2023-legislative-summary.pdf#page=10)